helm/logging-app-chart/templates/fluentbit/daemonset.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ .Values.fluentbit.name }}
+  namespace: {{ .Release.namespace }}
+  labels:
+    app: {{ .Values.fluentbit.name }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.fluentbit.name }}
+  template:
+    metadata:
+      annotations:
+        releasetime: {{ $.Release.Time }}
+      labels:
+        app: {{ .Values.fluentbit.name }}
+    spec:
+      tolerations:
+      # Tolerate master taint
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      serviceAccountName: {{ .Values.fluentbit.name }}
+      securityContext:
+        runAsUser: {{ .Values.fluentbit.userID }}
+        runAsGroup: {{ .Values.fluentbit.groupID }}
+      containers:
+      - name: {{ .Values.fluentbit.name }}
+        image: quay.io/giantswarm/fluent-bit:0.14.6
```

Comment:
Should the images be templated for the registry etc?

Like in https://github.com/giantswarm/fmt/blob/master/helm/image_templating.md.

Reply:
yeah good idea